docs/modules/reference/pages/service-assurance/monitors/DhcpMonitor.adoc

```diff
@@ -79,22 +79,28 @@ For Debian/Ubuntu, use `/var/lib/opennms/rrd/response`.
 
 .Configure a DHCP service in `poller-configuration.xml`
 
-Note that you must include the `monitor` section for each service in your definition.
-
 [source, xml]
 ----
 <service name="DHCP" interval="300000" user-defined="false" status="on">
- <parameter key="retry" value="2" />
- <parameter key="timeout" value="3000" />
- <parameter key="relayMode" value="false"/>
- <parameter key="extendedMode" value="false"/>
- <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response" />
- <parameter key="rrd-base-name" value="dhcp" />
- <parameter key="ds-name" value="dhcp" />
+ <parameter key="retry" value="2" /> <1>
+ <parameter key="timeout" value="3000" /> <2>
+ <parameter key="relayMode" value="false"/> <3>
+ <parameter key="extendedMode" value="false"/> <4>
+ <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response" /> <5>
+ <parameter key="rrd-base-name" value="dhcp" /> <6>
+ <parameter key="ds-name" value="dhcp" /> <7>
 </service>
 
-<monitor service="DHCP" class-name="org.opennms.netmgt.poller.monitors.DhcpMonitor"/>
+<monitor service="DHCP" class-name="org.opennms.netmgt.poller.monitors.DhcpMonitor"/> <8>
 ----
+<1> Number of attempts to test a service’s status
+<2> Timeout for the isReachable method, in milliseconds.
```

Comment:
Markdown isn't happy with the adoc format, the markdown gets semi lost when rendered [no gray background]. This will need to be resolved likely a different way.